ui/hooks/useTokenFiatAmount.js

```diff
@@ -32,7 +32,7 @@ export function useTokenFiatAmount(
   const conversionRate = useSelector(getConversionRate);
   const currentCurrency = useSelector(getCurrentCurrency);
   const userPrefersShownFiat = useSelector(getShouldShowFiat);
-  const showFiat = overrides.showFiat ?? userPrefersShownFiat;
+  const showFiat = overrides?.showFiat ?? userPrefersShownFiat;
```

Comment:
This file throws an error when getConversionRate is hardwired to return undefined, because it doesn't check if this is available before trying to compute values.